File: lecture_06_python/problem_set/sentimental_mario_more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")

    if height > 0 and height <= 8:
        break

for i in range(height):
    n = i + 1
    print(" " * (height - n) + "#" * n, end="  ")
    print("#" * n)

File: lecture_01_c/mario.c
#include <cs50.h>
#include <stdio.h>

void draw_bricks(const int width, const int height);

int main(void) {
    int w;
    int h;

    // Prompt user for positive integer
    do {
        w = get_int("enter width: ");
    } while (w < 1);
    do {
        h = get_int("enter height: ");
    } while (h < 1);

    draw_bricks(w, h);
}

void draw_bricks(const int width, const int height) {
    for (int i = 0; i < height; i++) {
        for (int j = 0; j < width; j++) {
            printf("#");
        }
        printf("\n");
    }
    printf("\n");
}
